CHANGELOG.md

```diff
@@ -3,7 +3,13 @@ This changelog documents the changes between release versions.
 
 ## [Unreleased]
 - Use separate schema files for each collection
-- Don't sample from collections that already have a schema
+- Changes to `update` CLI command:
+  - new default behaviour:
+    - attempt to use validator schema if available
+    - if no validator schema then sample documents from the collection
+  - don't sample from collections that already have a schema
+  - if no --sample-size given on command line, default sample size is 10
+  - new option --no-validator-schema to disable attempting to use validator schema
```

Comment:
Can we link to the PRs for these changelog entries?

Reply:
Done